src/SingleCellAnnotator.imjoy.html

```diff
@@ -627,40 +611,47 @@
       if(!sendData.discarded){
         if(this.current_image_data.location_changed){
           sendData.location = {};
-          sendData.location = this.current_image_data.location;
-          let noLocation = true;
-          for (loc in sendData.location){
-            if(sendData.location[loc] === null){
-              sendData.location[loc] = false
-            }
-            else{
-              noLocation = false;
-            }
-          }
-          if (noLocation && this.current_image_data.label_display_mode==='multiple'){
-            const anwser = await api.confirm("No location label selected, is that correct?")
-            if (!anwser){
-              return
+          if (this.current_task_name !== 'segmentation'){
```

Comment:
```suggestion
          if (!this.current_image_data.location){
```

Reply:
you are right. good you figured out a way